### PR TITLE
Update README with explanations and neater formatting.

### PR DESCRIPTION
**Pull request by alexnewmannn**
- Added explanation for all guidelines
- Updated definition of js- prefixes (would like input from everyone)
- Added media queries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,18 @@
 **[Mixins](#mixins)**  
 **[- Mixin Presentation](#mixin-presentation)**  
 **[- Mixin Use](#mixin-use)**  
+**[Media Queries](#media-query-usage)**
 
 ##File Naming
-File Names should be camelcased and, if necessary, platform should be separated with a hyphen.
+File Names should be camelcased and, if necessary, be preceded by platform and file type which `zshould be separated with a hyphen.
```
Comment (liam-swinney) [inline README.md]: zshould

File: README.md [hunk 2]
```diff
@@ -64,19 +78,25 @@ All properties should precede a semicolon.
...
 ```
 
-####Bad
+#####Bad
 ```less
 content: "";
 ```
 
+#####Why?
+Although there is no benefit to using either quotation single quotations should be used in LESS and Javascript leaving double quotations for HTML files.
```
Comment (liam-swinney) [inline README.md]: missing comma after 'either quotation'.  But add it allows for consistency.

Comment (alexnewmannn) [inline README.md]: Updated

**Comment by ahumphreys87** [comment]
:+1: lgtm

**Pull request merged**